examples/src/main/java/com/glegoux/grpc/examples/hello/HelloServer.java

```diff
@@ -1,13 +1,34 @@
 package com.glegoux.grpc.examples.hello;
 
+import com.glegoux.examples.hello.HelloGrpc;
+import com.glegoux.examples.hello.HelloReply;
+import com.glegoux.examples.hello.HelloRequest;
+import com.glegoux.grpc.GrpcServer;
+import com.glegoux.grpc.GrpcServerRunner;
 import com.glegoux.grpc.GrpcServerSimple;
+import com.google.common.collect.Lists;
+import io.grpc.BindableService;
+import io.grpc.stub.StreamObserver;
 
 import java.io.IOException;
+import java.util.List;
 
 public class HelloServer {
 
   public static void main(String[] args) throws IOException, InterruptedException {
-    new GrpcServerSimple().run("hello", args);
+    List<BindableService> services = Lists.newArrayList(new HelloImpl());
+    GrpcServer grpcServer = new GrpcServerSimple(services);
+    GrpcServerRunner.run("hello", args, grpcServer);
+  }
+
+  private static class HelloImpl extends HelloGrpc.HelloImplBase {
+
```

Comment:
we could later reorganize the structure and have a package service for thi class.

Reply:
For the moment, I am going just to extract the class to address your comment. But we could evolve the structure progressively with the evolution of the project to a target better structured. I think we can remain simple for the moment because the `hello` is simple. And see it later.

Reply:
sure!